features/user_can_create_invoice.feature

```diff
@@ -0,0 +1,13 @@
+Feature: User can create invoice
+
+  Background:
+    Given the following user exist
+      | email         | password |
+      | user@user.com | password |
+
+    And I am logged in as "user@user.com"
+    And I am on the invoices page
```

Comment:
once you have pulled in the upstream development, I need to see how you get to this page instead of going there directly

Reply:
added navigation to invoices page